CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ We use *breaking :warning:* word for marking changes that are not backward compa
 ### Fixed
 
 * [#3095](https://github.com/thanos-io/thanos/pull/3095) Rule: update manager when all rule files are removed.
+* [#3098](https://github.com/thanos-io/thanos/pull/3098) ui: Fix Block Viewer for Compactor and Store
```

Comment:
No need for this, as it's fixing the release directly. But that's ok I will adjust it.

Reply:
Thanks!